Tools/machines/pitzer-osc/pitzer_warpx.profile.example

```diff
@@ -0,0 +1,68 @@
+# please set your project account
+export proj=""  # change me!
+
+# remembers the location of this script
+export MY_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+if [ -z ${proj-} ]; then echo "WARNING: The 'proj' variable is not yet set in your $MY_PROFILE file! Please edit its line 2 to continue!"; return; fi
+
+module purge
+# load these modules for optimization (todo: make this compile)
+# module load cmake/3.25.2
+# module load intel/19.0.5
+# module load cuda/11.6.1
+# module load mvapich2-gdr/2.3.7
+# export CC=$(which icc)
+# export CXX=$(which icpc)
+# export FC=$(which ifort)
+
+# load these modules for stability
+module load cmake/3.25.2
+module load gnu/12.3.0
+module load cuda/12.3.0
+module load openmpi/4.1.5-hpcx
+export CC=$(which gcc)
+export CXX=$(which g++)
+export FC=$(which gfortran)
+
+export CUDACXX=$(which nvcc)
+export CUDAHOSTCXX=${CXX}
```

Comment:
We usually set those at the end of the profile script (see other examples, e.g. Perlmutter).